src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -936,6 +953,7 @@ impl GraphEditorModelWithNetwork {
     ( &self
     , cursor_style : &frp::Source<cursor::Style>
     , output_press : &frp::Source<NodeId>
+    , input_press  : &frp::Source<(NodeId,span_tree::Crumbs)>
```

Comment:
Use `EdgeTarget` instead of `(NodeId,span_tree::Crumbs)`. I undertand that logically it is not an "edge target" here, but this struct maybe should have a better name. Anyway a target (input or output) should be indicated by `EdgeTarget`.